lib/build.gradle

```diff
@@ -105,10 +105,16 @@ dependencies {
 	flexmarkCompileOnly 'com.vladsch.flexmark:flexmark-all:0.62.2'
 }
 
+configurations.named('testCompatKtLint0Dot48Dot1Implementation').configure {
+	extendsFrom(configurations.testImplementation, configurations.compatKtLint0Dot48Dot1CompileOnly)
```

Comment:
Regarding exposing the `compileOnly` libraries it makes perfect sense. I've had some thoughts about making the plugin export that automatically to the test implementation classpath, as it's likely that you'd want to refer to that version of the library as part of the test suite. Do you think it would make sense to automatically add a configuration called something like `compatXYZSpecific` that `compatXYZCompileOnly` and `testCompatXYZImplementation` would extend from? "Specific" is up for discussion.

Reply:
🤔 Having a `compatXYZSpecific` configuration would be quite flexible since everything is modular in nature. If you actually want a different library on the classpath, you simply don't need to add anything to `compatXYZSpecific`, but rather to the downstream configurations directly.

Reply:
@Sineaggi Do you have any thoughts about my info-dump here? :)

Reply:
@davidburstrom I had missed the `testCommonImplementation` configuration when looking thru the docs.

Also, I like the idea of the `compatXYZSpecific`.  `compatXYZCompileOnlyAndTestImplementation` maybe? :)

Reply:
I'll see what I can do :) Naming is hard...

Reply:
I've published version 0.4.0 of the plugin now, with support for `CompileAndTestOnly`. See https://github.com/davidburstrom/version-compatibility-gradle-plugin#compatibility-adapters.